Initium-Odp/war/odp/javascript/messager-impl.js

```diff
@@ -112,6 +112,9 @@ messager.onChatMessage = function(chatMessage)
 			chatMessage.message = "/me is a spammer. Kill him with fire!";
 		var meMode = chatMessage.message.toLowerCase().startsWith("/me ");
 		html+="<span class='chatMessage-text'>";
+		if(chatMessage.message.toLowerCase() == "/help") {
+			chatMessage.message = "<a href=\"#\" onclick=\"pagePopup('/odp/chatHelp.html')\">[Click here to view chat help]</a>"; 
```

Comment:
If you still want to do it this way I'll have to add it in my end. It's OK whatever you decide you want to do.